### ISSUE TITLE
Implement Modifier class which contains server adoptable draft.js modifiers

### ISSUE THREAD

**Issue by aight8** [webstronauts/draft-php#9]
from DraftModifier.js:
- [ ] replaceText
- [ ] insertText
- [ ] moveText
- [ ] splitBlock
- [ ] applyInlineStyle
- [ ] removeInlineStyle
- [ ] applyEntity

from AtomicBlockUtils.js:
- [ ] insertAtomicBlock

getRangesForDraftEntity.js
- [ ] getRangesForDraftEntity

**Comment by aight8** [comment]
@robinvdvleuten can you please make some comment's on this one too ;)

**Comment by robinvdvleuten** [comment]
@aight8 I agree with removing the SelectionState but as I see it in your proposal, we spread the SelectionClass to two separate classes right?

**Comment by aight8** [comment]
Good 👍 
Yeah, exactly. Sometimes you need a range, in other cases just a position (key + offset).

**Comment by aight8** [comment]
I removed SelectionState in the current PR.

I don't implemented the Modifier class directly for now. But I implemented similar methods in the ContentBlock class, which must be used anyway when there is a overall Modifier class which can modify across ContentBlock's. For now I see this Modifier class as an overkill. But maybe for the future.
I currently only see only ```Modifier::splitBlock``` as useful, since this could split blocks at soft newline characters etc.

See #13
